app/schemas/archive-format.json

```diff
@@ -0,0 +1,222 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://the-federation.info/specs/backup-restore#archive_format",
```

Comment:
One should host the exact document at that URL. We can host stuff on diaspora.software, perhaps a hostname pointing to some Github pages repo or just on the diasporafoundation.org server.